create_data_image.sh

```diff
@@ -1,25 +1,14 @@
 #!/bin/bash
 
-# Currently everything runs as root, but that is going to change.
-USER_UID=0
-USER_GID=0
-
-source ./su_command.sh
-
 TOTAL_SIZE=0
 
 echo "Gathering applications..."
-APPS=
-for app in apps/*
-do
-	if test -f $app
-	then
-		APPS="${APPS} $app"
-		SIZE=$(stat -Lc %s ${app})
+for app in apps/*; do
+	if test -f "$app";	then
```

Comment:
Is this a fix for app names that contain spaces? If so, please extract it into a separate commit.

Reply:
Done (#8)